modin/numpy/math.py

```diff
@@ -0,0 +1,276 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import numpy
+
+
+def absolute(
+    x, out=None, where=True, casting="same_kind", order="K", dtype=None, subok=True
+):
+    if hasattr(x, "_absolute"):
+        return x._absolute(
+            out=out, where=where, casting=casting, order=order, dtype=dtype, subok=subok
+        )
+    return numpy.absolute(
+        x, out=out, where=where, casting=casting, order=order, dtype=dtype, subok=subok
+    )
```

Comment:
```suggestion
    return getattr(x, "_absolute", lambda **kw: numpy.absolute(x, **kw))(
        out=out, where=where, casting=casting, order=order, dtype=dtype, subok=subok
    )
```
?
same for other functions here

Reply:
I mentioned this in a meeting with Rehan, reposting here for visibility: we decided that specifying a special attribute (like `MODIN_NP_ARRAY = True`) on the `modin.numpy.array` object, then using `hasattr` to check for that property would be preferable.

Reply:
I'm fine either way, I'm rather saying let's not duplicate the args - i.e. compute what to call with a `getattr` or `if hasattr()` or whatever, and then call once - repeating args two times increases chance for errors when developing.